integration/test/Testlib/ModService.hs

```diff
@@ -311,7 +311,7 @@ startBackend resource overrides services = do
           whenM (doesDirectoryExist path) $ removeDirectoryRecursive path
 
   let modifyEnv env = env {serviceMap = Map.insert resource.berDomain serviceMap env.serviceMap}
-      mkReq srv = do
+      checkServiceIsUp srv = do
```

Comment:
This function should return `pure True` for `Ngniz`

Reply:
😅